projects/dog-party.md

```diff
@@ -80,4 +80,38 @@ Take a few minutes and write out an action plan for the next few days.
   - Making sure classes are named appropriately
   - Organize your styles according to sections
   - Alphabetizing style properties
-  - Grouping classes together that share similar styles
\ No newline at end of file
+  - Grouping classes together that share similar styles
+
+### Eval Grading Rubric (20 minutes each)
+
+For this project, you get to be in charge of the eval!  This is a great opportunity to review another person's code and assess something that isn't your own.  This is a common thing you'll do when working on your group projects and when you have a job.  Assessing code and seeing what works and what could potentially be improved on is beneficial for you and your peers!  Here's a rubric with some questions to think about as your partner presents their project and you assess.
+
+Write out atleast three or more points of feedback in each section.  
+
+#### Design/Comp Recreation
+
+* How close does their site match the comp?  Are there small details that are missing?
+  - Pay attention to spacing, readability, sizing/positioning of elements
+* Is your partner's site responsive?  Are elements logically spaced as the window gets smaller?
+
+#### HTML Structure/Style
+
+[HTML Style Guide](https://github.com/turingschool-examples/html)
+
+* Take a look through the style guide and check if your partner's code follows these rules.
+* Are they using mostly semantic HTML? (staying away from nested divs, using correct elements, etc.)
+* Is the HTML structure clean and stears clear from unnecessary nesting?
+
+#### CSS Structure/Style
+
+[CSS Style Guide](https://github.com/turingschool-examples/css)
+
+* Take a look through the style guide and check if your partner's code follows these rules.
+* Are the styles organized in a logical way?  Are the properties for each class organized alphabetically or by similar groups (widths and heights, margins and paddings, etc.)
+* Do the class names clearly explain what elements they are meant to style?
+* Are classes that share similar styles grouped together?  If multiple classes share similar styles, could it be refactored into one class?
+* Are there instances where your partner is using `!important`?
+
+#### Terminology
```

Comment:
I think we maybe pull out this section as it could open the door for people to be mean?

Or we could maybe just try to make the questions more objective? - have them looks for specifics? How often are they using the word thing?